concourse/client/model.py

```diff
@@ -393,6 +395,12 @@ def name(self):
         return self.raw['name']
 
 
+@dataclass
+class PinnedVersion:
```

Comment:
not all resource versions are pull-request versions
--> define a base-class + add a specialised class for PRs